drivers/ad7746/Makefile

```diff
@@ -1 +1 @@
-include $(RIOTBASE)/Makefile.base
+include $(RIOTBASE)/drivers/driver_with_saul.mk
```

Comment:
If you follow this through, IMO this should include Makefile.base explicitly, not through driver_with_saul.mk.
E.g.:

```
include $(RIOTBASE)/drivers/driver_with_saul.mk
include $(RIOTBASE)/Makefile.base
```

... and drop the include in Makefile.base.

Reply:
Not sure I get why that would be better. `periph.mk`, `auto_init.inc.mk` or `application.inc.mk` are doing the same as `driver_with_saul.mk`.

Reply:
hmok